crates/turbopack-resolve/src/resolve_options_context.rs

```diff
@@ -72,6 +72,7 @@ pub struct ResolveOptionsContext {
     /// A list of plugins which get applied before (in the future) and after
```

Comment:
probably need to adjust comment bit, also we may want to rename `plugins` in this case?